README.md

```diff
@@ -1,2 +1,31 @@
-# Website
-This repo is work in progress for the official website of **[Kubesimplify](https://kubesimplify.github.io/website/)**
+<p align="center"><img width="192" img src="/static/branding/assets/png/icon/color.png" alt="Kubesimplify">
+
+<h1 align="center"><a href="https://kubesimplify.github.io/website/">Kubesimplify Website </a></h1>
+<p align="center"><i>The source code for Kubesimplify.com</i></p>
+
+Kubesimplfy's new website. This codebase is what runs on [Kubesimplify](https://kubesimplify.github.io/website/). For new developers getting started, run the following in your terminal:
+
```

Comment:
```suggestion

Fork the repo by clicking the fork button. This will create a copy of the repo in your account, changes done here will not be shown in the main repo. 
 
```